tests/runfiles/linux.run

```diff
@@ -104,9 +104,9 @@ tests = ['zfs_create_001_pos', 'zfs_create_002_pos', 'zfs_create_003_pos',
 # zfs_destroy_012_pos - busy mountpoint behavior
 # zfs_destroy_013_neg - busy mountpoint behavior
 [tests/functional/cli_root/zfs_destroy]
-tests = ['zfs_destroy_002_pos', 'zfs_destroy_003_pos', 'zfs_destroy_004_pos',
```

Comment:
Actually, one last minor thing.  The `# zfs_destroy_001_pos - busy mountpoint behavior` comment should be removed above since the test has been enabled.